ui/package.json

```diff
@@ -84,14 +87,34 @@
     "wiredep": "~2.2.0"
   },
   "engines": {
-    "node": ">=6.10.x"
+    "node": ">=6.10.x",
+    "yarn": ">= 1.0.0"
   },
   "scripts": {
     "init": "npm install",
     "install": "bower install",
-    "test": "./node_modules/.bin/karma start --single-run --browsers Chrome"
+    "test": "./node_modules/.bin/karma start --single-run --browsers Chrome",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'bower_components', 'junction') } catch (e) { }\""
   },
   "dependencies": {
-    "jasmine-core": "^2.5.2"
+    "jasmine-core": "^2.5.2",
+    "@bower_components/angular": "angular/bower-angular#1.6.7",
```

Comment:
Is it possible to use angular directly rather than from bower? Same applies to other places as well. @rajikaimal @saisankargochhayat @agentmilindu

Reply:
@rehrumesh  Yes, and for some bower modules, we must do this! Some don't have `package.json` files as these are Bower Packages, and we have to find the relevant NPMs for them.

Reply:
@pranayrobo It seems the packages here are very popular ones, thus won't be a big headache to find the relevant NPM packages. You can use `angular-animate` instead of `angular/bower-angular-animate`, angular-aria` instead of `angular/bower-angular-aria`, `d3` instead of `mbostock-bower/d3-bower`, etc. 

Simply, go to npmjs.com and search for the packages, like "angular-cookies", etc. You'll be able to find the relevant NPM packages.

Let's change them and merge the PR. The rest Looks good to me.